### DISCUSSION TITLE
Contextual model for visual information processing

### DISCUSSION THREAD

**Discussion by iluhakhurtin** [AGI-course/agi-course.github.io#5] [Ideas]
The PhD project I am working on relates to the contextual model of the visual information processing. We try to support it with what is happening in the primary visual cortex of an alive brain of some mammalians. We suppose the project is based on the principles an AGI can use. This is about the stage of perception what prof. Pei Wang explained in the lectures but also can be applicable to the further layers (out of the PhD scope).
The first stage is done and the article is under review and has been [pre-printed](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4240434). 
Thus, I would like to present the contextual approach and show the model results. Is it applicable?

**Comment by patham9** [comment]
Yes, your work is clearly related to the field: feel free to take this as an opportunity to work on your perception model, and to obtain and summarize related results. For the latter, both a writing, and open source prototype will work.

**Reply by iluhakhurtin** [reply]
Thank you!
